.vscode/settings.json

```diff
@@ -4,5 +4,8 @@
         "editor.defaultFormatter": "JohnnyMorganz.stylua"
     },
     "stylua.targetReleaseVersion": "v0.17.0",
-    "Lua.runtime.version": "Lua 5.1"
+    "Lua.runtime.version": "Lua 5.1",
+    "Lua.workspace.library": [
+        "DCS_API_defs.lua"
```

Comment:
What does this do?

Reply:
First one includes the GetJSON() function.
The settings.json, it includes the DCS API definitions.